basic-examples/bind-constants/README.md

```diff
@@ -0,0 +1,79 @@
+# Binding Repeated Values
+
+## Workflow Description
+
+This workflow demonstrates how to use the Arcaflow builtin function `bindConstants()` to repeat a set of input values with each item in your input list using a foreach loop.
+
+### Subworkflow Description
+
+The subworkflow uses a plugin to print the value of the `name` input.  The `loop_id` and `ratio` input values are bundled with this plugin output as this subworkflow's output.

```

Comment:
I would be inclined to add a comment that, if this were a "real" workflow instead of just an example, the plugin would operate on the data and produce some output, and that the template plugin is used to simulate that.

Reply:
did it